### PR TITLE
feat: upgrade @babel/core

### PR DESCRIPTION
**Pull request by 9renpoto**
closes https://github.com/uber/react-digraph/issues/186

upgrade `@babel` packages

### CODE REVIEW

File: webpack.prod.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 const webpack = require('webpack');
-const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
```
Comment (9renpoto) [inline webpack.prod.js:3]: refs: https://webpack.js.org/guides/production/#minification

**Review by ajbogh — APPROVED** [review]

**Comment by ajbogh** [comment]
Thanks for the PR.

**Comment by 9renpoto** [comment]
why close this ? 🤔

**Comment by ajbogh** [comment]
Sorry, wrong button. I was commenting and accidentally hit the Close with comment button.

**Comment by 9renpoto** [comment]
thank you review :)

**Pull request merged**